src/wolfgame.py

```diff
@@ -2784,39 +2781,38 @@ def reaper(cli, gameid):
                     cli.msg(chan, messages["channel_idle_warning"].format(", ".join(x)))
                 msg_targets = [p for p in to_warn_pm if p in pl]
                 mass_privmsg(cli, msg_targets, messages["player_idle_warning"].format(chan), privmsg=True)
-            for dcedplayer in list(var.DISCONNECTED.keys()):
-                acc, hostmask, timeofdc, what = var.DISCONNECTED[dcedplayer]
+            for dcedplayer, (timeofdc, what) in list(var.DISCONNECTED.items()):
```

Comment:
No need to cast into a list if we're not mutating the variable.

Reply:
it **is** being mutated. del_player removes the user from var.DISCONNECTED

Reply:
Oh, alright then